src/pages/Login/SignUp.js

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import SignInput from '../../components/Sign/SignInput';
+import SignBtn from '../../components/Sign/SignBtn';
+import inputData_list from '../../components/Sign/InputData';
+import './signup.scss';
+
+class SignUp extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      userInfo: {},
+    };
+  }
+
+  getValue = e => {
+    const { name, value } = e.target;
+    this.setState({
+      userInfo: { ...this.state.userInfo, [name]: value },
+    });
+  };
+
+  // 비밀번호 확인 유효성 검사 기능 구현
+  pwValidationCheck = e => {
+    const { value } = e.target;
+    let password = this.state.userInfo.password;
+    if (!value) {
+      console.log('');
+    } else if (password !== value) {
+      alert('비밀번호가 일치하지 않습니다.');
+    }
```

Comment:
- if 문의 처음 조건은 아무런 동작을 하지 않으므로, 그냥 두번째 조건만 검사해주셔도 될 것 같습니다.

```suggestion
    if (password !== value) {
      alert('비밀번호가 일치하지 않습니다.');
    }
```

Reply:
확인! 수정완료했습니다 :)